### PR TITLE
Update JWT dependency to 2.1

### PR DESCRIPTION
**Pull request by stefanrendevski**
This is my first pull request, so let me know if I'm missing anything.

This pull request is based on #182

I would have been more permissible with the version, but the JWT token used in specs is hardcoded, and different versions of JWT generate a different token, which would cause the specs to fail. I do not know how to extract the hard-coded token and generate one with a runtime-determined version.

### CODE REVIEW

**Comment by knodi** [comment]
@msach22 - Did travis just have a temporary glitch?   Maybe you can click a re-run button or something?

**Comment by msach22** [comment]
Thanks for submitting a PR @stefanrendevski!

@knodi It looks like the tests for failing for some versions of ruby - I've gone ahead and re-ran them and will share the results 😄

**Comment by thedexican** [comment]
bump ^ is this gonna get fixed / merged?

File: opentok.gemspec [hunk 1]
```diff
@@ -28,5 +28,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency "addressable", "~> 2.3" #  2.3.0 <= version < 3.0.0
   spec.add_dependency "httparty", "~> 0.15.5"
   spec.add_dependency "activesupport", ">= 2.0"
-  spec.add_dependency "jwt", "~> 1.5.6"
+  spec.add_dependency "jwt", "~> 2.1"
```
Comment (msach22) [inline opentok.gemspec]: Can we change this to `>= 1.5.6` ? This way it won't be a breaking change and folks would be able to upgrade to whichever version?

**Comment by jimedelstein** [comment]
Hello!  Another bump, is this gonna get fixed / merged?

**Comment by jeffswartz** [comment]
Thanks, @stefanrendevski for your PR. We merged in a similar PR to update dependencies (https://github.com/opentok/OpenTok-Ruby-SDK/pull/192), and we will close this one.

**Pull request closed without merging**